mosaic2/include/grid2.inc

```diff
@@ -894,20 +386,8 @@ subroutine get_grid_cell_vertices_2D(component, tile, lonb, latb, domain)
   type(FmsNetcdfFile_t)  :: tilefileobj
   logical :: valid_types = .false.
 
-  select type(lonb)
-  type is (real(r4_kind))
-    select type(latb)
-    type is (real(r4_kind))
-      call get_grid_size_for_one_tile(component, tile, nlon, nlat)
-      valid_types = .true.
-    end select
-  type is (real(r8_kind))
-    select type(latb)
-    type is (real(r8_kind))
-      call get_grid_size_for_one_tile(component, tile, nlon, nlat)
-      valid_types = .true.
-    end select
-  end select
+  call get_grid_size_for_one_tile(component, tile, nlon, nlat)
+  valid_types = .true.
```

Comment:
you can get rid of the valid_types variable, it was only added as a easy way to error out if the types don't match up.

Reply:
good catch, this one must have slipped